sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -2270,6 +2283,24 @@ def from_dict(cls, data):
             confidence=data.get("confidence", None),
         )
 
+    def get_words(self, **kwargs):  # pylint: disable=unused-argument
+        # type: (Any) -> Iterable[DocumentWord]
+        """Get the words found in the spans of this AnalyzedDocument.
+
+        :return: iterable[DocumentWord]
+        :rtype: iterable[DocumentWord]
+        """
+        return _get_children(self, ["word"], ["word"], True)
```

Comment:
nit: I haven't looked at _get_children yet, but wondering why we pass the same argument twice. Might help to pass like keywords so you can see the param name